ivy_tests/test_ivy/test_frontends/test_tensorflow/test_raw_ops.py

```diff
@@ -66,6 +66,34 @@ def test_tensorflow_Acosh(  # NOQA
     )
 
 
+# Angle
+@handle_frontend_test(
+    fn_tree="tensorflow.raw_ops.Angle",
+    dtype_and_x=helpers.dtype_and_values(
+        available_dtypes=helpers.get_dtypes("float"),
```

Comment:
I think this should be of "complex" dtype, as this function is supposed to work on complex arrays.